interface-spec.md

```diff
@@ -108,11 +108,17 @@ Triple is an alias of Quad.
 
 ### DataFactory
 
+For default values of the instance properties, see the individual [interface
+definitions](#data-interfaces)
+
 **Methods:**
 
 - `IRI .iri(String iri)` returns a new instance of IRI.
 - `BlankNode .blankNode()` returns a new instance of BlankNode.
-- `Literal .literal(String value, String language, String datatype)` returns a new instance of Literal.
+- Literal .literal(String value, [String languageOrDatatype]) returns a new
+  instance of Literal. If languageOrDatatype is an IRI, then a NamedNode is
```

Comment:
Even in your syntax I don't think you will often use relative IRIs for **datatype** of the Literal term type. This issue doesn't talk about IRIs which denote `s p o g` but IRIs used for datatypes, most often `xsd:` https://www.w3.org/TR/rdf11-concepts/#section-Datatypes